packages/grammar/README.md

```diff
@@ -0,0 +1,22 @@
+# @wordpress/grammar
+
+This library provides a grammar defining what kinds of content can be represented inside a WordPress content.
```

Comment:
Provides not just a grammar, but also a parser. I'm wondering which module name is better. Also, we should consider the fact that there won't be a single parser (see #6831 and friends).

Reply:
my vote for the name is "spec-parser" since that's what it is. it's the grammar as defined in the PEG and the auto-generated parser from that specification.